Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: ensure_pre_commit
```

Comment:
Does this move us to how our other terraform-* repos are setup?

Reply:
Yep, I use this in other modules.